Anastasiya Santalova/hw1.html

```diff
@@ -0,0 +1,281 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1, maximum-scale=1">
+
+    <title>Cubes / Santalova</title>
+
+    <link rel="stylesheet" href="hw1.css">
+  </head>
+  <body>
+
+  <div class="widget">
+
+    <div class="row">
+
+      <div class="weather weather-skyblue">
+
+        <div class="weather__block_visible"> 
```

Comment:
You have a lot of `div` and `span` elements that are not semantic. It's not critical but sometimes it's better to use semantic tags like `header`, `main`, `section`, `footer` etc.

Reply:
My widget has no header and footer, but I've changed some tags. Thank you for comments!